src/rc_libretro.c

```diff
@@ -51,6 +51,11 @@ static const rc_disallowed_setting_t _rc_disallowed_dolphin_settings[] = {
   { NULL, NULL }
 };
 
+static const rc_disallowed_setting_t _rc_disallowed_dosbox_pure_settings[] = {
+  { "dosbox_pure_strict_mode", "false" },
```

Comment:
You should add a test to `test_rc_libretro` for this.

Reply:
Added a test in the latest commit